### PR TITLE
Updating logs pipeline diagram

### PR DESCRIPTION
**Pull request by kaileyhaynes**
Colorized and updated the logs pipeline diagram to be dark-mode friendly, show what is the customer data, and separate the line between log forwarder and log backend.

### CODE REVIEW

File: src/content/docs/logs/log-management/ui-data/drop-data-drop-filter-rules.mdx [hunk 1]
```diff
@@ -33,10 +33,10 @@ A drop filter rule matches data based on a query. When triggered, the drop filte
 
 This creates an explicit demarcation between the logs being forwarded from your domain and the data that New Relic collects. Since the data removed by the drop filter rule doesn't reach our backend, it cannot be queried: the data is gone and cannot be restored.
 
-![New Relic Logs - Architecture - Drop filter](./images/NRDB2.png "New Relic Logs - Architecture - Drop filter")
+![New Relic Logs - Architecture - Drop filter](./images/ingest-pipeline.svg "New Relic Logs - Architecture - Drop filter")
 
 <figcaption>
-  During the ingestion process, log data can be parsed, transformed, or dropped before being stored.
+  During the ingestion process, customer log data can be parsed, transformed, or dropped before stored to New Relic's database.
```
Comment (urbiz-nr) [inline src/content/docs/logs/log-management/ui-data/drop-data-drop-filter-rules.mdx:39]: Stored in...

**Review by urbiz-nr — DISMISSED** [review]
Nice diagram! Left a comment, up to you!

**Comment by paperclypse** [comment]
I think the diagram looks fantastic, but it looks like the image is running into the sidebar. Maybe because it's an SVG image?

Screen Shot 2021-06-24 at 11 33 28 AM

**Review by paperclypse — DISMISSED** [review]
Ready to merge in once you decide what to do about the image :)

**Comment by kaileyhaynes** [comment]
@paperclypse I'll merge when either you or another confirms that it works with the png :)

File: src/content/docs/logs/log-management/ui-data/drop-data-drop-filter-rules.mdx [hunk 2]
```diff
@@ -33,10 +33,10 @@ A drop filter rule matches data based on a query. When triggered, the drop filte
 
 This creates an explicit demarcation between the logs being forwarded from your domain and the data that New Relic collects. Since the data removed by the drop filter rule doesn't reach our backend, it cannot be queried: the data is gone and cannot be restored.
 
-![New Relic Logs - Architecture - Drop filter](./images/NRDB2.png "New Relic Logs - Architecture - Drop filter")
+![New Relic Logs - Architecture - Drop filter](./images/ingest-pipeline.png "New Relic Logs - Architecture - Drop filter")
 
 <figcaption>
-  During the ingestion process, log data can be parsed, transformed, or dropped before being stored.
+  During the ingestion process, customer log data can be parsed, transformed, or dropped before stored in New Relic's database.
```
Comment (urbiz-nr) [inline src/content/docs/logs/log-management/ui-data/drop-data-drop-filter-rules.mdx:39]: "Being" is missing - "... dropped before being stored in NR's database".

**Comment by urbiz-nr** [comment]
Diagram looks good to me.

**Review by urbiz-nr — CHANGES_REQUESTED** [review]
Please check comment.

**Review by paperclypse — APPROVED** [review]
Looks good to me!

**Pull request merged**